tailwind.css

```diff
@@ -0,0 +1,17 @@
+@import "tailwindcss" prefix(tw);
+
+
+/* body {
+    font-family: "Roboto", Corbel, Avenir, "Lucida Grande", "Lucida Sans", sans-serif;
+    color: #0C3C26;
+    line-height: 1.6;
+    -moz-osx-font-smoothing: grayscale;
+} */
+
+:root {
+    --font-family: "Roboto", Corbel, Avenir, "Lucida Grande", "Lucida Sans", sans-serif;
+    --color-primary: #0C3C26;
+    --color-secondary: #F2F2F2;
+    --line-height: 1.6;
+    --font-smoothing: grayscale;
+}
```

Comment:
Not sure about this, these styles come from the `oc.css` file and presumably are there to override specific styles provided by Bootstrap. I could be wrong, but I would think switching to css variables on the root wouldn't override those styles anymore?